doc/customizing/cloning-and-building.md

```diff
@@ -21,7 +21,7 @@ If you run into trouble or want more explanation, read on.
 TerriaJS can be built and run on almost any macOS, Linux, or Windows system. The following are required to build TerriaJS:
 
 -   The Bash command shell. On macOS or Linux you almost certainly already have this. On Windows, you can easily get it by installing [Git for Windows](https://gitforwindows.org/). In the instructions below, we assume you're using a Bash command prompt.
--   [Node.js](https://nodejs.org) v14.0 or later. You can check your node version by running `node --version` on the command-line.
+-   [Node.js](https://nodejs.org) v16.0 or later. You can check your node version by running `node --version` on the command-line.
```

Comment:
I'm not able to build locally with `v18` or above, for now should this just say `v16`

Reply:
`node-sass` v6 used by terriajs doesn't work on newer node versions, but I don't see a point in upgrading it to a newer version as there is an initiative to replace it with dart-sass